weaviate/schema/crud_schema.py

```diff
@@ -764,6 +769,72 @@ def _create_classes_with_primitives(self, schema_classes_list: list) -> None:
         for weaviate_class in schema_classes_list:
             self._create_class_with_primitives(weaviate_class)
 
+    def create_class_tenants(self, class_name: str, tenants: Union[dict, str]) -> None:
+        """
+        Create classes tenants in Weaviate.
+
+        Parameters
+        ----------
+        tenants : dict or str
+            Class as a Python dict, or the path to a JSON file, or the URL of a JSON file.
+
+        Examples
+        --------
+        >>> tenants = [
+        ...    { "name": "Tenant1" },
+        ...    { "name": "Tenant2" }
+        ... ]
+        >>> client.schema.create_tenants(tenants)
+
+        If you have your tenants json file saved in the './schema/tenants.json' you can create it
+        directly from the file.
+
+        >>> client.schema.create_tenants('./schema/tenants.json')
+
+        Raises
+        ------
+        TypeError
+            If the 'tenants' is neither a string nor a dict.
+        ValueError
+            If 'tenants' can not be converted.
+        requests.ConnectionError
+            If the network connection to Weaviate fails.
+        weaviate.UnexpectedStatusCodeException
+            If Weaviate reports a non-OK status.
+        weaviate.SchemaValidationException
+            If the 'schema_class' could not be validated against the standard format.
+        """
+
+        loaded_tenants = _get_dict_from_list_object(tenants)
+        self._create_class_tenants(class_name, loaded_tenants)
```

Comment:
why not add the content of `self._create_class_tenants(class_name, loaded_tenants)` directly here. The `self._create_class_tenants(class_name, loaded_tenants)` should not be called anywhere else, right? This will same some time because the function will not be added to the stack.

Reply:
Good point, fixed that!